core/src/main/scala/org/apache/spark/api/python/PythonRDD.scala

```diff
@@ -313,6 +313,7 @@ private object SpecialLengths {
   val PYTHON_EXCEPTION_THROWN = -2
   val TIMING_DATA = -3
   val END_OF_STREAM = -4
+  val NULL = -5
```

Comment:
Also if this is a core python change, there should be unit tests that capture this change.